src/pages/login/loginStyle.tsx

```diff
@@ -0,0 +1,72 @@
+import { makeStyles } from "@material-ui/core/styles";
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    height: "100vh",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    backgroundImage: `linear-gradient(to top, #30cfd0 0%, #330867 100%)`,
+    "& .MuiContainer-root": {
+      backgroundColor: "#f1f2f4",
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "center",
+      padding: "20px 0 40px 0",
+      borderRadius: "30px",
+    },
+    "& h2": {
```

Comment:
You don't need to do this when using Typography 
just pass the class to the component